### PR TITLE
Generate and commit TypeScript declaration files

### PR DESCRIPTION
**Pull request by hparra**
Improve devex for TS users.

## Types of changes

- [x] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [x] I have signed the [Adobe Open Source CLA](http://opensource.adobe.com/cla.html).
- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the **CONTRIBUTING** document.
- [ ] I have added tests to cover my changes.
- [x] All new and existing tests passed.

### CODE REVIEW

**Comment by hparra** [comment]
Build failures don't seem to be related to changes. main ok?

**Comment by shazron** [comment]
test failure here: wait for https://github.com/adobe/aio-lib-core-logging/pull/60 to be reviewed and pulled in, then merge

**Review by moritzraho — APPROVED** [review]

File: package.json [hunk 1]
```diff
@@ -19,10 +21,11 @@
   },
   "dependencies": {
     "debug": "^4.1.1",
-    "winston": "^3.2.1"
+    "winston": "3.12"
```
Comment (moritzraho) [inline package.json:24]: this won't be needed anymore once #60 is merged

Comment (shazron) [reply package.json:24]: @hparra please revert [b6fdb40](https://github.com/adobe/aio-lib-core-logging/pull/57/commits/b6fdb403415377972fea64f31d1e245d8714c6cf) so we can pull this in. I've merged the fix from master

Comment (hparra) [reply package.json]: @shazron done

**Review by shazron — APPROVED** [review]

**Pull request merged**